### DISCUSSION TITLE
喝水

### DISCUSSION THREAD

**Discussion by juliaouo** [1101-GS4719-C/Python-Community-GS4719-C#1398] [General]
從以前開始我就是一個不怎麼喜歡喝水的人，常常會在感到口渴的時候才會喝一兩口，只有在喝飲料時才會不自覺一直喝，很常拿在手裡莫名其妙就沒了XD上了大學後更加自由，目前是一天一杯手搖的程度，不過感覺好像不太健康，想要做點改變。不知道大家有沒有什麼強迫自己喝水的方法，因為真的是坐在電腦前時間不知不覺就過去了才發現好像一整天也沒喝到一口水。不知道有人有同樣的問題嗎？

**Comment by tower20021202** [comment]
雖然我也很常坐在電腦桌前，但我依然堅持每日要喝至少2000cc的水，維持身體的健康，如果怕忘記的話，不妨每小時設一個鬧鐘提醒自己順便喝點水

**Comment by brian431** [comment]
那就把一大瓶水壺擺在自己面前ㄅ。看到他空掉了就快去裝，看到他有點滿就喝一喝。這樣應該就解決問題了。

**Comment by Tzuhsuannnnn** [comment]
我會買可以提著的水壺然後拿在手上，常常看到就會無意識喝一下，很快就可以喝完一罐水，推薦這個方法給妳！

**Comment by Miinteaa** [comment]
可以考慮購入有自帶吸管的環保杯！實測過用吸管喝水的話就會不知不覺中喝掉很多水，也許可以改善你的困擾：）！

**Comment by TW-Maverick** [comment]
有沒有一種可能你不喜歡喝水，其實是懶得去裝水？我也會發生類似的事情，但倘若長時間久坐、或者座椅太硬，會造成「梨狀肌症候群」的發生。如何改善，我分享自己的經驗就是每隔1~1.5hr讓自己走動一下，不要因久坐而讓肌肉過於緊繃，可以試著去上廁所或裝水，順便可以強迫自己喝水。

**Comment by Evening218** [comment]
我最近也正在強迫自己喝水然後盡量不要喝手搖!動機也只是因為買了一個新的水壺~不過我認為要培養喝水習慣的話,首先真的是要堅決戒掉其他飲料,並且隨時在手邊準備好水,這樣應該就可以慢慢習慣喝水了

**Comment by zilinng** [comment]
我鼓勵自己喝水的方式是買很多可愛的杯子，因為看到好看的杯子就會想要用他，所以可以喝比較多水，其次就是用超大水壺裝水逼自己一定要喝完，不僅可以很準確的知道自己喝的水量，也不會因為懶得裝水導致沒喝夠。

**Comment by ziihui** [comment]
有吸管的水壺真的會讓你愛上喝水!!!因為感覺很像在喝飲料那樣，也推薦手機可以下載提醒你喝水的APP，看到自己每天達標也會很有成就感喔~

**Comment by shuxian12** [comment]
我自己非常喜歡喝水，以前國中的時候可以一節課喝掉一罐水，非常快樂，但就是可能需要每節下課都去上廁所，但現在也是很喜歡喝水，飲料也是，壓力大的時候一天也會喝一杯手搖，但每次喝完都覺得好渴，就會又喝了很多水。
然後也因為飲水機太遠了，所以為了處理這件事，買了一個2.5公升的大水壺，放在書桌上，好快樂(´･ω･`)

**Comment by pigeon9975** [comment]
之前捐血的時候發現自己的血抽不太起來，才發現自己真的喝太少水了，也不是因為不喜歡喝水，只是單純覺得既然沒有渴就不用喝，看來這種喝水習慣好像不太健康qq我感覺我也需要提醒喝水的app

**Comment by alicia0725** [comment]
以前我也是只會記得喝飲料不記得喝水，後來改用環保杯一定要喝完裡面的水，才能喝飲料，可以提高喝水的意願！也推薦你設定時的鬧鐘離開電腦桌前起來走走再喝水！

**Comment by poweichen00** [comment]
我以前是超愛喝飲料的人，後來我買了一個大水壺上面有刻幾點要喝到哪，後來成功戒掉了飲料。

**Comment by Alexgns** [comment]
上大學後越來越少喝水，後來便隨時攜帶一個水瓶或水杯，以便隨時補充水分，在上課、圖書館或宿舍都可以設置喝水提醒，以確保定期喝水，有一個APP:Plant Nanny可以用來提醒

**Comment by zephyrxin01** [comment]
高鈉飲食、不喝水愛喝飲料的同學們請真的要注意多多補充水分!!小心腎結石年輕化，前陣子我朋友腹部不明原因很痛，結果檢查後發現有輕微的腎結石QQ 就被醫生告誡要多喝水，改變飲食習慣了。

**Comment by Chiuhsianglin** [comment]
從小我也不喜歡喝水，喜歡喝手搖杯，但上了大學買了一個有吸管的水壺後，水就喝越來越多了，所以我也很推薦有一個有吸管的水壺！確診後我也了解到了喝水有多麼重要。

**Comment by 1104030360** [comment]
我會使用iphone的內建程式來監控自已每天的飲水量，或是直接買超大水壺裝滿放在旁邊，這樣就會強迫自己喝水

**Comment by aa103967622** [comment]
就像你容易被飲料的特別包裝所吸引，你的水壺也可以隨時提醒你喝水，去買一個你喜歡造型的水壺，放在你隨時看得見或是隨手可拿的地方，喝水就會像呼吸一樣簡單，當你的口渴已經用健康的水所滿足，自然就不會去超商或手搖店買那些不健康的飲品了。

**Comment by xiang305** [comment]
之前在youtube上看到了結石的手術影片，看起來很痛，就覺得自己該養成喝水的習慣了。我的方法是都把水壺放在隨手可以拿到的地方，出門也會帶著，想到就拿起來喝，慢慢的就養成習慣了，現在一天至少都能喝2公升以上。

**Comment by Elaine2899** [comment]
我發現剛換水壺的時候，大概可以抱持兩周：每天會喝兩瓶水壺的量。上次我換的新水壺是770CC，另外，我覺得吸管式的水壺比較會讓我一直喝水，因為我懶得一直開蓋子然後抬頭喝水，如果是吸管的話我就可以邊抄筆記邊喝水，所以我那陣子每天至少有1500，最近也至少會喝一瓶(770)喝得多的話可以到3瓶水的量，因此找到適合自己的水壺或許也可以增加自己的喝水量哦!

**Comment by bestbanana295** [comment]
我覺得喝水對於人體來說非常重要，畢竟身體大部分是油水組成的沒有喝水的話頭常常會覺得昏沉，所以每天補足喝水量是哩常重要的。

**Comment by MaxChi870** [comment]
我習慣桌上放個杯子，裝白開水也好，泡杯咖啡或紅茶也行，讓自己習慣喝水的動作，空了就重裝，也趁機讓自己別一直坐著。

**Comment by Alex11392** [comment]
可以就放一壺水在桌上，強迫自己過一小時喝一次

**Comment by 110302001** [comment]
同樣推薦有吸管的水壺!因為感覺有咬著東西可以喝一下再停一下，我之前有一陣子在減肥，每天都要強迫自己喝2000cc,但我也是不喜歡喝水，後來換個水壺後改善很多~

**Comment by BOSENCHANG2004** [comment]
人體大約有60%是水組成的，幾乎所有器官和細胞都需要水來工作。水就像身體裡的潤滑劑和搬運工，水能幫助腎臟過濾血液、排出毒素（尿素、尿酸等），如果水喝得太少，可能會導致腎結石、尿道感染，甚至慢性腎病。有些人覺得餓，其實只是「口渴」。水能幫助產生飽足感，也能促進身體燃燒熱量。想減肥的人，喝水是很好的小幫手！

**Comment by lilipop6405** [comment]
我好像比較沒有這個問題，我都是想喝的時候就喝，通常都不會忘記喝，而且常常覺得出房間去裝水好麻煩，所以我甚至買了一個大水壺放在家裡，去裝水就用大水壺裝，這樣可以減少出門的次數。

**Comment by yuuu3957** [comment]
我是小時候的時候超級不喜歡喝水(這邊小時候的範圍大概到國中)，大概是一天都喝不完半瓶水壺的程度(500ml的水壺)。
但後面高中就開始正常喝水。
現在我是逼著我自己每天都要喝1500ml-2000ml的量，一開始很痛苦，但現在堅持一年多了，喝不夠還很容易口渴。

**Comment by clairechien0627** [comment]
我也是不怎麼喜歡喝水的人，頂多就是夏天到了真的很熱才會勉強喝一口，帶水壺就跟帶裝飾品一樣的，看到水就真的不會想喝的程度，就算喝也是只喝一點點，所以作為替代我都會自己泡茶喝，或是買牛奶呵。

**Comment by yuku-huang** [comment]
你從以前就不喜歡喝水，只有口渴或喝飲料時才會喝幾口。上大學後因為更自由，一天會喝一杯手搖，但也意識到這樣不健康。你常常坐在電腦前整天沒喝水，想改變這個習慣。你可以設定定時提醒，每小時喝水一次；把水壺放在桌上增加可見度；把喝水和日常動作綁在一起，例如滑手機前先喝水。

**Comment by yurukung** [comment]
我超喜歡喝水，常常覺得口乾舌燥，但好像聽中醫的講法，會有這樣的情況是因為上火😟

**Comment by Welly940504** [comment]
我完全懂你那種「水喝不下去、但飲料喝超快」的感覺（笑）！那種放一杯水在旁邊可以放到天荒地老，但手搖一拿到直接半小時內喝光的情況我也超有感。上大學之後沒人管，要買飲料超方便，一天一杯變成常態也很自然

**Comment by jaychang1027** [comment]
我自己也是不怎麼愛喝水的人，但每天還是會強迫自己至少喝2000cc，平常練球的時候，就會一直喝水，很容易就達成了，但撇除掉練球的時間，自己喝水的頻率就會下降，會很想喝飲料，一天一杯已經成為常態，特別是練完球之後會很想來一杯冰涼的。

**Comment by yunnnnchieh** [comment]
我也是很容易忘記喝水的人，常常會立目標說要維持喝水的習慣，但容易忘記帶水壺，或是在上課在外面奔波就會喝飲料，只有在去運動之後感覺到明顯的口渴的時候才會攝取比較多的水分，感覺為了健康著想應該要有意識的提醒自己喝水，夏天也會比較不容易中暑

**Comment by shrshu38** [comment]
就像你容易被飲料的特別包裝所吸引,你的水壺也可 以隨時提醒你喝水,去買一個你喜歡造型的水壺,放 在你隨時看得見或是隨手可拿的地方,喝水就會像呼 吸一樣簡單,當你的口渴已經用健康的水所滿足,自 然就不會去超商或手搖店買那些不健康的飲品了。

**Comment by 12388833** [comment]
可以強迫自己一天至少要喝2000cc的水，也就是大概裝三次水壺的量，當作一種任務，強迫自己完成，我個人是對飲料還好，不會特別想去喝手搖杯，可能也是從小在家的習慣吧

**Comment by LINSHIHMIN** [comment]
可以買個喜歡的水壺隨身帶著 有在瓶身設計什麼時候應該喝到多少水的水壺 感覺能提醒自己該喝水了！雖然那種水壺通常都蠻大一個的 但如果長時間在電腦前待著 就不用一直去裝水～ 而且如果喜歡喝手搖飲的話 也推薦有吸管的水壺！覺得水沒味道不喜歡喝 也能加點檸檬片之類的 希望能從強迫自己喝水 變成習慣甚至喜歡喝水！

**Comment by ghost** [comment]
我之前也常常會忘了喝水，甚至有時候忘了帶水壺就索性不喝了，說不定一天還喝不到1000cc。也是前陣子好奇才查到，原來自己一天至少得喝2000cc以上的水，所以現在我都會規定自己在一個時間段內最少要喝多少水。

**Comment by zhen-yuu** [comment]
我之前是從無糖茶開始欸，或者是無糖的氣泡水也很棒，後來就慢慢接受喝水了，而且個人覺得冰水比較好喝

**Comment by yue0903ZZ** [comment]
把一天當中最想喝手搖飲的時段，用無糖茶或氣泡水取代，慢慢減少糖分攝取，也能增加喝水量。
找到最適合自己的方式，畢竟喝水是維持健康最基本的事，對精神、皮膚和身體機能都超重要的！

**Comment by ichenx2** [comment]
我原本也是不愛喝水的人，也跟你一樣超愛喝飲料，但後來我換了一個有吸管的水壺之後發現自己一天可以喝很多水，不知道是不是因為用吸管會有一種像在喝飲料的錯覺，喝起來特別順🤣，有一陣子換回一般水壺，喝水量真的馬上減少…所以我覺得可以試試看換成有吸管的水壺，說不定也對你有用。

**Comment by alice20040127** [comment]
沒有不喜歡喝水，但也不太常會喝水，原因是因為我沒有辦法吸收那麼多，常常喝一杯水一下下就要跑洗手間了，大一的時候曾經有一段時間超級愛喝水，然後就是一整天一直在跑洗手間，如果是待在宿舍或教學樓都沒關係，可是一旦出去到外面就會很麻煩，找不到洗手間超崩潰。

**Comment by a414414** [comment]
我覺得喝水很重要，水是人體代謝非常重要的要素，而且我覺得喝水也能讓我保持清醒，我自己也不討厭喝水，一天至少會喝到2000以上，雖然會一直跑廁所，但是我覺得去上一下廁所不會怎樣啦，健康最重要。況且，我也看過有人天天喝手搖飲，都不喝水，結果年紀輕輕(可能20歲?)就腎結石，我不想這樣，所以我會乖乖喝水，喝爆多水。

**Comment by wayne935** [comment]
我其實蠻喜歡喝水的，但是很多時候會忘記要喝，聽說規律喝水應該才是對的，等到自己覺得渴的時候才喝可能會比較不好。我有時候甚至會因為喝茶喝太多，而忘記要喝水，這樣好像也不太好。
